packages/edit-site/src/components/site-icon/index.js

```diff
@@ -41,7 +41,7 @@ function SiteIcon( { className } ) {
 	) : (
 		<Icon
 			className="edit-site-site-icon__icon"
-			size="32px"
+			size="48px"
```

Comment:
Feels a bit large when its the WordPress icon. 

As-is: 

https://user-images.githubusercontent.com/1813435/223741778-693f0081-8cab-4f44-aa77-bbb5bd294b38.mp4

But if we use `40px`, and `0.65` for scale we have this: 

https://user-images.githubusercontent.com/1813435/223742852-8a3c5a2f-1d74-4af1-b30d-ff194133f5fe.mp4

Reply:
Although (with the suggestion above) the site icon is perhaps a bit big.

https://user-images.githubusercontent.com/1813435/223743492-5e3b2d5b-7e59-40fe-8f1f-0ea7ca72d029.mp4